bin/reth/src/node/events.rs

```diff
@@ -43,18 +43,38 @@ impl NodeState {
                 self.current_checkpoint = checkpoint.unwrap_or_default();
 
                 if notable {
-                    info!(target: "reth::cli", stage = %stage_id, from = ?checkpoint, "Executing stage");
+                    if let Some(checkpoint) = checkpoint {
+                        info!(
```

Comment:
Can't we just use `self.current_checkpoint` that is assigned a default value up above?

Reply:
yeah we can, fixed